Podverse/DownloadsTableViewController.swift

```diff
@@ -102,19 +104,21 @@ extension DownloadsTableViewController:UITableViewDelegate, UITableViewDataSourc
         
         let downloadingEpisode = DownloadingEpisodeList.shared.downloadingEpisodes[indexPath.row]
         
-        if downloadingEpisode.taskResumeData != nil {
-            if !PVDownloader.shared.shouldDownload() {
-                self.showAllowCellularDataAlert()
-            }
-            pvDownloader.resumeDownloadingEpisode(downloadingEpisode: downloadingEpisode)
-        } else if downloadingEpisode.downloadComplete == true {
+        if downloadingEpisode.downloadComplete == true {
             let moc = CoreDataHelper.createMOCForThread(threadType: .mainThread)
             if let mediaUrl = downloadingEpisode.mediaUrl {
                 let episode = CoreDataHelper.retrieveExistingOrCreateNewEpisode(mediaUrlString: mediaUrl, moc: moc)
                 let playerHistoryItem = playerHistoryManager.convertEpisodeToPlayerHistoryItem(episode: episode)
                 goToNowPlaying()
                 pvMediaPlayer.loadPlayerHistoryItem(item: playerHistoryItem)
             }
+        } else if downloadingEpisode.taskResumeData != nil {
+            if !PVDownloader.shared.shouldDownload() {
+                self.showAllowCellularDataAlert()
```

Comment:
The way this is written I think you show the alert about allowing cellular data but then we resumeDownloading any way on the next line. Or am I missing something?